test/integration/middleware/with-node.js-apis/test/index.test.ts

```diff
@@ -0,0 +1,114 @@
+/* eslint-env jest */
+
+import {
+  fetchViaHTTP,
+  findPort,
+  killApp,
+  launchApp,
+  nextBuild,
+  waitFor,
+} from 'next-test-utils'
+import { join } from 'path'
+
+jest.setTimeout(1000 * 60 * 2)
+
+const unsupportedFunctions = [
+  'setImmediate',
+  'clearImmediate',
+  'structuredClone',
```

Comment:
I think `structuredClone` is supported in the edge runtime.

Reply:
allowed, thanks!